webui/src/components/SearchForm.vue

```diff
@@ -0,0 +1,109 @@
+<template>
+    <div class="input-form input-search">
+        <input type="text"
+               v-model="text"
+               placeholder="Find people"
+               @input="searchSubjects(text)"
+               @blur="hideSearchedSubject"/>
+        <div class="action-button" @click="searchSubjects(text)">
+            <SearchIcon/>
+        </div>
+
+        <transition name="fade">
+            <div class="suggested-subjects-list" v-show="searchedSubjectShoved">
+                <div v-for="item in searchedSubjects"
+                     class="suggested-subjects-item"
+                     @click="selectInterestingSubject(item)">{{item.displayName}}
+                </div>
+            </div>
+        </transition>
+    </div>
+</template>
+
+<script>
+    import SearchIcon from './../components/Icons/SearchIcon'
+
+    export default {
+        name: "SearchForm",
+
+        components: {SearchIcon},
+
+        data() {
+            return {
+                searchedSubjectShoved: false,
+            }
+        },
+
+        computed: {
+            text: {
+                get() {
+                    return this.$store.state.user.searchValue;
+                },
+
+                set(changed) {
+                    this.$store.commit('CHANGE_SEARCH_VALUE', changed);
+                }
+            },
+
+            searchedSubjects: {
+                get() {
+                    return this.$store.state.user.searchedSubjectsList;
+                }
+            }
+        },
+
+        methods: {
+            showSearchedSubject(query) {
+                console.log(query);
+                if (query) {
+                    this.searchedSubjectShoved = true;
+                }
+            },
+
+            hideSearchedSubject() {
+                this.searchedSubjectShoved = false;
```

Comment:
IMHO we don't need the searchedSubjectShoved field at all. It can be done through computed property, like this:
```
            displayFoundSubjects: {
                get() {
                    return this.$store.state.user.searchValue.length > 0;
                }
            }
```